### ISSUE TITLE
Line break in help text

### ISSUE THREAD

**Issue by jycr** [tkambler/electron-preferences#205]
Is it possible to have line break in help description?

Following config does not work:

```json
{
  "fields": [
    {
      "label": "Working directory",
      "key": "workingDir",
      "type": "file",
      "help": "Lorem ipsum dolor sit amet.\nVestibulum ante ipsum primis in faucibus."
    },
  ]
}
```

<img width="387" alt="image" src="https://github.com/snapcrunch/electron-preferences/assets/51145/259ef7a5-b515-48a9-a499-18780cf9917b">

**Comment by pvrobays** [comment]
Hi @jycr 

This is not possible by default. We don't parse the text as HTML. We just pass the text to react which renders it within a span element.

You could create a workaround where you add your custom CSS which adds the `white-space: pre;` attribute to the `.help` CSS class. This is the class that's used on the help span element.
I didn't try this, but I assume in that case your `\n` character will be interpreted as a new line.

**Comment by pvrobays** [comment]
I'll mark this as `won't fix`. Can you verify the workaround?

**Comment by lacymorrow** [comment]
Does `<br />` not work?

**Comment by pvrobays** [comment]
> Does `<br />` not work?

I don't think so. We just pass the config variable on to the html tag in React. If you want to parse the variable as HTML, you need to use the `dangerouslySetInnerHTML` property.
It could be an option to make this work as the input is coming from devs, not from end-users so you can 'trust' the input. However, I think using the preformatted html tag is a better fit in this case.
